lib/organization-manager.js

```diff
@@ -166,6 +166,44 @@ OrganizationManager.prototype.addMembers = function (data, session) {
     });
 };
 
+// Add members to the given team with updating users status.
+//
+// data     - Object,
+// session  - String, the session key.
+//
+// Returns a promise.
+OrganizationManager.prototype.addStudents = function (data, session) {
+
+    assert.string(data.orgId, 'organization');
+    assert.string(data.teamId, 'team');
+    data.memberIds && assert.arrayOfString(data.memberIds, 'memberIds');
+    data.memberNames && assert.arrayOfString(data.memberNames, 'memberNames');
+    assert.string(session, 'session_id');
+
+    return this.signed('addStudents', data, {
+        'session_id' : session
+    });
+};
+
+// Add members to the given team with updating users status.
+//
+// data     - Object,
+// session  - String, the session key.
+//
+// Returns a promise.
+OrganizationManager.prototype.addTeachers = function (data, session) {
+
+    assert.string(data.orgId, 'organization');
+    assert.string(data.teamId, 'team');
+    data.memberIds && assert.arrayOfString(data.memberIds, 'memberIds');
+    data.memberNames && assert.arrayOfString(data.memberNames, 'memberNames');
+    assert.string(session, 'session_id');
+
+    return this.signed('addTeachers', data, {
+        'session_id' : session
```

Comment:
You do not need the 'session_id' for signed request:

return this.signed('addTeachers', data, {})